src/Elyon.Fastly.Api.DomainServices/InfoSessionFollowUpService.cs

```diff
@@ -31,6 +33,8 @@ public class InfoSessionFollowUpService : BaseService, IInfoSessionFollowUpServi
         private readonly IInfoSessionFollowUpRepository _infoSessionFollowUpRepository;
         private readonly IOrganizationsRepository _organizationsRepository;
         private readonly IEmailSenderService _emailSenderService;
+        private const int _companyOrganizationId = 82000;
+        private const int _smeOrganizationId = 99990;
```

Comment:
is this _smeOrganizationId an OrganizationId or OrganizationTypeId? because in the code below you compare it with organizationtypeid